dags/pegs-challenge-submission-dag.py

```diff
@@ -1,26 +1,34 @@
+import uuid
+import os
 from datetime import datetime
 
 from airflow.decorators import dag, task
 from airflow.models import Param
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
 
 from orca.services.nextflowtower import NextflowTowerHook
 from orca.services.nextflowtower.models import LaunchInfo
 from orca.services.synapse import SynapseHook
 
+UUID = uuid.uuid4()
+REGION_NAME = "us-east-1"
+BUCKET_NAME = "pegs-challenge-project-tower-scratch"
+FILE_NAME = f"submissions_{UUID}.csv"
```

Comment:
nit: Just use UUID directly in this:
```suggestion
FILE_NAME = f"submissions_{uuid.uuid4()}.csv"
```

Reply:
Thanks Bryan, just need to clarify something: @BWMac was this written as an environment variable in case it changes from `uuid4` in the future, or can it just be folded into the string?

Reply:
I think I wrote it that way so I could reuse the `UUID` constant, but then I ended up only using it once

Reply:
Got it, thanks for the quick reply